modules/apps/object/object-test/src/testFunctional/macros/ObjectCustomViews.macro

```diff
@@ -186,12 +186,46 @@ definition {
 		}
 	}
 
+	macro clickOnBackInFilter {
+		Click(locator1 = "ObjectCustomViews#BACK_FILTER");
+	}
+
+	macro clickOnFilterOnEntries {
+		Click(
+			key_buttonName = "Filter",
+			locator1 = "ObjectCustomViews#DROPDOWN_BUTTON_ON_OBJECT_VIEW");
+	}
+
```

Comment:
In the future, we probably can avoid macros like this if we're using tasks to explain what the test is doing.